src/e2e/index.ts

```diff
@@ -1,64 +1,83 @@
-import characterItemMacroTests, {
-  key as characterItemMacroKey,
-  options as characterItemMacroOptions
-} from './actor/createItemMacro.test';
-
-import characterTests, {
-  key as characterKey,
-  options as characterOptions
-} from './actor/character.e2e.test.js';
-
+/**
+ * @file Orchestration for our Quench tests
```

Comment:
Managed to merge everything, including my `container_fix` PR (#313) into the linting branch; ran all the tests and they all came out OK at least!